### PR TITLE
feat: add docs for noise suppression plugin

### PR DESCRIPTION
**Pull request by vishaldhull09**

### CODE REVIEW

File: docs/javascript/v2/plugins/noise-suppression.mdx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+isSupported can be used to check if the browser is supported or not.
+This accepts sampling rate as a parameter
+- `sampleRate` - Number of audio samples taken per second
+
+It will return True/False based on browser.
+
+```jsx
+if (noiseSuppressionPlugin.isSupported(sampleRate)) {
+  console.log('Platform/sampleRate is supported');
```
Comment (sagargaonkar) [inline docs/javascript/v2/plugins/noise-suppression.mdx:67]: We should return what exactly is not supported

File: docs/javascript/v2/plugins/noise-suppression.mdx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+} else {
+  console.log('Platform/sampleRate is not supported');
+}
+```
+### Init(Optional)
+
+Init is used to load model of background noise suppression for the first time.
+It takes on an average 52 milliseconds.<br/>
```
Comment (sagargaonkar) [inline docs/javascript/v2/plugins/noise-suppression.mdx:75]: Make it "...50..."

File: docs/javascript/v2/plugins/noise-suppression.mdx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    console.log('noise suppression failure - ', isNoiseSuppressed, err);
+  }
+
+  return <Button onClick={toggleNoiseSuppression} />;
+}
+```
+
+</Tab>
+
+### Set background noise suppression
```
Comment (sagargaonkar) [inline docs/javascript/v2/plugins/noise-suppression.mdx:144]: Should this be "Enable/disable background noise suppression" ?

File: docs/javascript/v2/plugins/noise-suppression.mdx [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+**Get the 100ms NoiseSuppression Package**
+
+```bash
+npm install --save @100mslive/hms-noise-suppression
+```
+
+## Import plugin
+
+```jsx
+import HMSNoiseSuppressionPlugin from '@100mslive/hms-noise-suppression';
```
Comment (raviteja83) [inline docs/javascript/v2/plugins/noise-suppression.mdx:42]: { HMSNoiseSuppressionPlugin }

File: docs/javascript/v2/plugins/noise-suppression.mdx [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+
+```jsx
+import { hmsActions } from './hms';
+
+const pluginSupport = hmsActions.validateAudioPluginSupport(
+        noiseSuppressionPlugin
+      );
+if (pluginSupport.isSupported) {
+      console.log('Plugin is supported');
+} else{
```
Comment (raviteja83) [inline docs/javascript/v2/plugins/noise-suppression.mdx:70]: format properly

File: docs/javascript/v2/plugins/noise-suppression.mdx [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+      console.log('Plugin is not supported');
```
Comment (raviteja83) [inline docs/javascript/v2/plugins/noise-suppression.mdx:71]: keep only one console log.

File: docs/javascript/v2/plugins/noise-suppression.mdx [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+
+```jsx
+import {
+  useHMSStore,
+  useHMSActions,
+  selectIsLocalAudioPluginPresent
+} from '@100mslive/hms-video-react';
+
+function NoiseSuppression() {
+  const isNoiseSuppressed = useHMSStore(selectIsLocalAudioPluginPresent(noiseSuppressionPlugin.getName());
```
Comment (raviteja83) [inline docs/javascript/v2/plugins/noise-suppression.mdx:126]: add how noiseSuppressionPlugin is created here as well. People will copy the code

File: docs/javascript/v2/plugins/noise-suppression.mdx [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+
+### Enable/disable background noise suppression
+
+The function `setEnabled` can be used to enable/disable background noise suppression without removing the plugin from audio track.
+It accepts boolean as a parameter.
+
+```jsx
+try{
+  noiseSuppressionPlugin.setEnabled(true); // true/false
+}catch(err){
```
Comment (raviteja83) [inline docs/javascript/v2/plugins/noise-suppression.mdx:156]: format

File: docs/javascript/v2/plugins/noise-suppression.mdx (hunk 7 above)
Comment (A-KVerma) [reply docs/javascript/v2/plugins/noise-suppression.mdx:126]: This is the format we are following for VB also, having a separate section for instantiante plugin and add plugin.

Comment (A-KVerma) [reply docs/javascript/v2/plugins/noise-suppression.mdx:125]: We have already added a section for Instantiate plugin above, and that instance is used everywhere

**Review by raviteja83 — APPROVED** [review]

**Review by triptu — APPROVED** [review]

File: docs/javascript/v2/plugins/noise-suppression.mdx [hunk 9]
```diff
@@ -0,0 +1,162 @@
+---
+title: Noise Suppression
+nav: 7.1
+---
+
+## Introduction
```
Comment (triptu) [inline docs/javascript/v2/plugins/noise-suppression.mdx:6]: can we put a beta notic(check vb doc page)

**Review by triptu — APPROVED** [review]

**Pull request merged**